src/engraving/rw/write/twrite.cpp

```diff
@@ -2983,12 +2983,7 @@ void TWrite::write(const TremoloSingleChord* item, XmlWriter& xml, WriteContext&
         return;
     }
 
-    if (ctx.clipboardmode()) {
-        xml.startElement(item);
-    } else {
-        // for compatible reason
-        xml.startElement(TConv::toXml(ElementType::TREMOLO).ascii());
```

Comment:
This is not necessary anymore, now that the file format version number has been updated to 440